pulp_2to3_migration/app/pre_migration.py

```diff
@@ -103,7 +104,7 @@ def delete_removed_pulp2_content(content_model):
                 pulp2_id__in=content_ids_to_delete
             ).delete()
 
-    batch_size = 100
+    batch_size = settings.CONTENT_PREMIGRATION_BATCH_SIZE or DEFAULT_BATCH_SIZE
```

Comment:
settings.get(CONTENT_PREMIGRATION_BATCH_SIZE, DEFAULT_BATCH_SIZE)

Reply:
I didn't use get on purpose.
I expect the option always to be present since it's a part of the plugin default settings. My concern is that it can be set to None or 0. And in this case `get` doesn't help.

Reply:
Maybe stronger validation is needed to ensure that it's a positive integer.

Reply:
imo that's an overkill :D i can't recall us doing such validation on settings. I think the way you have it now is good enough